markdown/dbaas.md

```diff
@@ -334,12 +398,58 @@ curl -X POST "http://localhost/v1/management/DBaaS/PXCClusters/GetCredentials" \
 
 API endpoint used in this step: [GetPXCClusterCredentials](ref:getpxcclustercredentials)
 
+### Create a PSMDB Cluster
+
+The PSMDB Create endpoint can also set defaults so, creating a PSMDB cluster can be made with a request like this:
+
+```bash
+curl -X POST "http://localhost/v1/management/DBaaS/PSMDBCluster/Create" \
+    -H "accept: application/json" \
+    -H "authorization: Basic YWRtaW46YWRtaW4=" \
+    -H "Content-Type: application/json" \
+    -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"expose\": true}"
+```
+
+#### Request fields
+
+```json
+{
+  "kubernetes_cluster_name": "string",
+  "name": "string",
+  "params": {
+    "cluster_size": 0,
+    "replicaset": {
+      "compute_resources": {
+        "cpu_m": 0,
+        "memory_bytes": "string"
```

Comment:
As this is defining request fields, isn't it confusing the reader by `string`, which might be read as the data type where all other fields aren't? Would it be better to define the request fields by data type for clarity?

Also, I don't see an example response, which is that the other examples in this doc use. Is that intentional?

Reply:
Added a note explaining why it is a string.
The response in empty with the corresponding HTTP status code.